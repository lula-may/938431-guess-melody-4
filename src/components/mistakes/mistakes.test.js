import React from "react";
import renderer from "react-test-renderer";
import Mistakes from "./mistakes.jsx";

describe(`Mistakes Component`, () => {
  it(`should render correctly 0 mistakes`, () => {
    const tree = renderer.create(
        <Mistakes
          count={0}
        />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it(`should render correctly 3 mistakes`, () => {
    const tree = renderer.create(
        <Mistakes
          count={3}
        />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
