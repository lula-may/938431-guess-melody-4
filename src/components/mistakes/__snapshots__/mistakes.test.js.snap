// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mistakes Component should render correctly 0 mistakes 1`] = `
<div
  className="game__mistakes"
/>
`;

exports[`Mistakes Component should render correctly 3 mistakes 1`] = `
<div
  className="game__mistakes"
>
  <div
    className="wrong"
  />
  <div
    className="wrong"
  />
  <div
    className="wrong"
  />
</div>
`;
